lib/MusicBrainz/Server/Entity/URL/Tsutaya.pm

```diff
@@ -0,0 +1,21 @@
+package MusicBrainz::Server::Entity::URL::Tsutaya;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { 'TSUTAYA' }
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT AND LICENSE
+
+Copyright (C) 2023 MetaBrainz Foundation
```

Comment:
@julian45, since this is your first contribution and this line is copied from another file, I have to make sure that it is intentional.

This line is assigning copyright for this contribution to the MetaBrainz Foundation, which is the preferred way as it allows us to enforce the license effectively when needed.

But it would be totally fine too if you personally preferred to add your own name (real or nickname) on another line to have explicit attribution beyond the git log.

In either case, you remain the copyright holder of your contribution.

Reply:
Would [this](https://github.com/metabrainz/musicbrainz-server/pull/2807/commits/9f5b869ea69a78569e7fa77f9528cf056608daed) work?

Reply:
Not really, commit https://github.com/metabrainz/musicbrainz-server/pull/2807/commits/9f5b869ea69a78569e7fa77f9528cf056608daed doesn’t follow the current format (when optionally specifying your name in the code):

```
Copyright (C) 2023 MetaBrainz Foundation
Copyright (C) 2023 <Your Name>
```

Sorry if my above comment wrongly let you think that I required you to make a change here, I did not, I just gave you the legal information.

You can either remove the line you just added, or follow the above template.

It is perfectly fine not to have a second line of Copyright notice. This is what most contributors do anyway, including me even when I was just volunteering. Note that you are already credited for your contributions through the git log.

Reply:
Got it, thank you for clarifying. This has been taken care of in 57a972f.

Reply:
You linked to a commit about cleanup. The “Drafted by” line is still there.

Reply:
I definitely wrote a line that fixed this earlier this morning, but it looks like it didn't get pushed in the moment. I guess it may have gotten lost between making changes in both my local copy and through GitHub. 4e6029abfa should have it. Sorry about that!